src/app.js

```diff
@@ -3,6 +3,13 @@ import { render } from 'preact';
 import { Header } from './components/Header.js';
 import { MainPage } from './pages/MainPage.js';
 
+const version = '1.0.2';
```

Comment:
We should automate the version in rollup, I can add this

Reply:
Made a separate issue for this: https://github.com/Code-for-All/lockdown/issues/82